### PR TITLE
fix: typo using utxo type

### PR DESCRIPTION
**Pull request by r4mmer**
### Acceptance Criteria
- token field is called `tokenId` on `Utxo` type
- allow deserializing `PartialTxInputData` with no signatures

### Security Checklist
- [ ] Make sure you do not include new dependencies in the project unless strictly necessary and do not include dev-dependencies as production ones. More dependencies increase the possibility of one of them being hijacked and affecting us.

### CODE REVIEW

File: src/models/partial_tx.ts [hunk 1]
```diff
@@ -596,7 +596,7 @@ export class PartialTxInputData {
    */
   addSignatures(serialized: string) {
     const arr = serialized.split('|');
-    if (arr.length < 3 || arr[0] != 'PartialTxInputData' || arr[1] !== this.hash) {
+    if (arr.length < 2 || arr[0] != 'PartialTxInputData' || arr[1] !== this.hash) {
```
Comment (pedroferreira1) [inline src/models/partial_tx.ts:599]: Don't we have the `PartialTxInputData` string, the hash and the signatures? So we need the array at least with length 3, right?

Comment (tuliomir) [inline src/models/partial_tx.ts:599]: Two comments about this line:

Why did you change the amount of elements in this `serialized` array? Is this related to a specific change?

And second, I think it would be better to use the string constant already there:
```suggestion
    if (arr.length < 2 || arr[0] != PartialTxInputDataPrefix || arr[1] !== this.hash) {
```

**Review by tuliomir — CHANGES_REQUESTED** [review]

File: src/models/partial_tx.ts (hunk 1 above)
Comment (r4mmer) [reply src/models/partial_tx.ts:599]: There is the case of a participant with no inputs requesting his signatures.
The input data part would be empty so we can either not return the signatures to the user (stating that the input data is empty) or allow the "no signatures" case and it seemed best to just have a noop case when the user "signature" does not contain any input data.

**Review by pedroferreira1 — APPROVED** [review]

**Review by tuliomir — APPROVED** [review]
